### DISCUSSION TITLE
UserFlows Beta :: Are We Stable Enough?

### DISCUSSION THREAD

**Discussion by visualsitemaps** [xyflow/xyflow#2354] [Show and tell]
Folks, after weeks and weeks of toil and +500 bug fixes.. we **_think_** we finally have a stable version! ( doing a hand-sign of the cross )

>>> [**Beta v1**](https://userflows.s.visualsitemaps.com/) <<<

We would be so grateful if you could kick our tires..and post any bugs **[here](https://docs.google.com/forms/d/e/1FAIpQLSfycoVwKCEKko_cLhtNDnGHDIAMEtMe3sGGEkYnDSnISpU9ZQ/viewform?usp=sf_link)** with a detailed screencast.

tip: try remaking this flow: ![.](https://i.imgur.com/Dd3wvL1.png)

thanks in advance!

**Comment by moklick** [comment]
wow! Thanks for sharing. You really pushed React Flow to its limits 💯

**Reply by visualsitemaps** [reply]
what is life.. when you cant push it to it's limits right?

**Comment by visualsitemaps** [comment]
and forgive me forgetting my manners.. THANK YOU FOR MAKING THIS HAPPEN! 
![.](https://i.imgur.com/EAWhGlT.gif)

**Comment by sumitngupta** [comment]
The interaction flow is fantastic
